templates/metrics.html

```diff
@@ -9,18 +9,16 @@
 {% block content %}
 <div class="superAbout">
     {% if not request.user_agent.is_mobile %}
-        {% include '_sidebar.html' with whichPage='metrics' title="Metrics" heTitle="מדדים" %}
+        {% include '_sidebar.html' with whichPage='metrics' %}
     {% endif %}
```

Comment:
Have you removed the `title` and `heTitle` from all the instances of `{% include '_sidebar.html %}`?

If so, maybe remove the rendered `<h1>` divs from `templates/_sidebar.html`

Reply:
No it still displays title and heTitle for mobile.